### PR TITLE
Add bot-base/scan-tool-bot

### PR DESCRIPTION
**Pull request by deptyped**

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
It seems to be offline?

**Comment by deptyped** [comment]
@KnorpelSenf It works for me. Can you check again? Is the bot not responding at all?

**Comment by roj1512** [comment]
It has been working for me, too.

**Comment by KnorpelSenf** [comment]
Checked again, now it is working. Previously, it did not respond at all.

**Review by KnorpelSenf — APPROVED** [review]
Awesome stuff!

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [Satont/twitch-notifier](https://github.com/Satont/twitch-notifier) - Get notified when a Twitch broadcast starts, ends or updates. [ᴜsᴇ](https://t.me/TwiNotifyBot)
 - [KnorpelSenf/link-preview-bot](https://github.com/KnorpelSenf/link-preview-bot) - Generate link previews for all links in messages. [ᴜsᴇ](https://t.me/linkpreviewbot)
 - [dcdunkan/syntax-highlighter-bot](https://github.com/dcdunkan/syntax-highlighter-bot) - Syntax highlighting for code blocks in messages. [ᴜsᴇ](https://t.me/syntaxybot)
+- [bot-base/scan-tool-bot](https://github.com/bot-base/scan-tool-bot) - A bot that scans QR codes with a camera. [ᴜsᴇ](https://t.me/ScanToolBot)
```
Comment (roj1512) [inline README.md:32]:
```suggestion
- [bot-base/scan-tool-bot](https://github.com/bot-base/scan-tool-bot) - Scan QR codes with your camera. [ᴜsᴇ](https://t.me/ScanToolBot)
```

**Review by roj1512 — CHANGES_REQUESTED** [review]

**Review by roj1512 — APPROVED** [review]

**Pull request merged**